metagpt/actions/project_management_an.py

```diff
@@ -54,16 +98,60 @@
     key="Shared Knowledge",
     expected_type=str,
     instruction="Detail any shared knowledge, like common utility functions or configuration variables.",
-    example="'game.py' contains functions shared across the project.",
+    example="`game.py` contains functions shared across the project.",
+)
+
+INCREMENTAL_SHARED_KNOWLEDGE = ActionNode(
+    key="Incremental Shared Knowledge",
+    expected_type=str,
+    instruction="Document any new shared knowledge generated during incremental development. This includes common "
+    "utility functions, configuration variables, or any information vital for team collaboration.",
+    example="`new_module.py` introduces shared utility functions for improved code reusability.",
 )
 
+REFINED_SHARED_KNOWLEDGE = ActionNode(
+    key="Refined Shared Knowledge",
+    expected_type=str,
+    instruction="Update and expand shared knowledge to reflect any new elements introduced. This includes common "
+    "utility functions, configuration variables for team collaboration. Retain content that is not related to "
+    "incremental development but important for consistency and clarity.",
+    example="`new_module.py` enhances shared utility functions for improved code reusability and collaboration.",
+)
+
+
 ANYTHING_UNCLEAR_PM = ActionNode(
     key="Anything UNCLEAR",
     expected_type=str,
     instruction="Mention any unclear aspects in the project management context and try to clarify them.",
     example="Clarification needed on how to start and initialize third-party libraries.",
 )
 
+INC_PM_CONTEXT = """
```

Comment:
These INC/MERGE_XX_CONTEXT not used?

Reply:
They are not in use, I am about to delete them